### PR TITLE
Czech translation of sample letters on blog

### PR DESCRIPTION
**Pull request by skvrnami**
I added Czech translation to the blog posts related to sample letters.

I also fixed a typo in the English "Sample letter for rectification requests as per Art. 16 GDPR" and deleted two sentence which (I believe) should not be there as they concern "deleting data" and specification of which data should be deleted (which doesn't make sense as this letter is supposed to be about rectification). I think these two sentences are an artifact from copying & pasting similar paragraphs from sample letter for erasure requests.

### CODE REVIEW

**Comment by mal-tee** [comment]
Hi @skvrnami, can you help us with the questions and remarks regarding the translation itself? Would be greatly appreciated.

We can then later on sort out the technical stuff by ourselves.

File: content/cz/blog/sample-letter-gdpr-rectification-request/index.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+{
+    "title": "Vzorový dopis pro žádost o opravu podle čl. 16 GDPR",
+    "slug": "sample-letter-gdpr-rectification-request",
+    "date": "2018-05-16T22:06:12+02:00",
+    "last_edited": "2020-10-11T01:28:42",
+    "type": "blog",
+    "description": "Pokud firma má uložené chybné (nebo nekompletní) osobní údaje týkající se Vás, podle Obecného nařízení o ochraně osobních údajů (GDPR) máte právo požadovat neodkladnou opravu takových chybných dat. Nabízíme Vám vzorový dopis, s pomocí kterého se můžete domáhat svých práv.",
+    "featured_image": "gdpr-rectification-request-letter",
+    "tags": [ "gdpr", "rectificacion request", "letter", "sample", "article 16" ],
+    "authors": [ "baltpeter" ],
```
[the remark this answers is not in the document]
Comment (skvrnami) [reply content/cz/blog/sample-letter-gdpr-rectification-request/index.md:10]: There is no need to list my name here.

**Comment by skvrnami** [comment]
Hi @mal-tee, I hope everything is ok now. I translated the file names and image description.

**Comment by mal-tee** [comment]
Thank you very much, we can take it from here. :)

**Comment by zner0L** [comment]
I moved this into #1053, so I’ll close it here. Thank you so much for your contributions!

**Pull request closed without merging**